src/main/app/claims/models/claim.ts

```diff
@@ -238,13 +238,27 @@ export class Claim {
     if (this.eligibleForCCJAfterBreachedSettlement) {
       statuses.push({ status: ClaimStatus.ELIGIBLE_FOR_CCJ_AFTER_BREACHED_SETTLEMENT })
     }
+
     if (!this.moneyReceivedOn || (!this.moneyReceivedOn && !this.countyCourtJudgmentRequestedAt)) {
-      statuses.push({ status: ClaimStatus.PAID_IN_FULL_ELIGIBLE })
+      if (this.isOfferSubmitted() || this.isResponseSubmitted() || (this.response && (this.response as FullAdmissionResponse).paymentIntention.paymentOption !== PaymentOption.IMMEDIATELY)
```

Comment:
If `this.response && (this.response as FullAdmissionResponse).paymentIntention.paymentOption !== PaymentOption.IMMEDIATELY` were true, then the earlier `this.isResponseSubmitted()` would also have been true, and so the former would never be tested

Reply:
Yes that first check is no longer needed thanks

Reply:
I need this.isResponseSubmitted() for non-admissions journeys, as without it this error is thrown:
`TypeError: Cannot read property 'paymentOption' of undefined`
So a check has to be done beforehand so it passes before reaching the 'FullAdmissionResponse' check.

Reply:
Sounds like you want `isResponseSubmitted() && blah blah blah` then

Reply:
Note: from discussions we established that `isResponseSubmitted()` returns false if the claimant has _also_ responded; it is a "is the claim in the 'response submitted' state" query rather than "has a response been submitted for the claim" query.